docs/source/tutorials/prompt.rst

```diff
@@ -1,10 +1,26 @@
 Prompt
 ============
+
+.. raw:: html
+
+   <div style="display: flex; justify-content: flex-end; align-items: center; margin-top: -50px;">
```

Comment:
- make the image in the colab smaller
- too much details, it looks like a total repeat on the tutorial. Cut down the text